tests/KubernetesClient.Tests/TokenFileAuth.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using k8s.Authentication;
+using Xunit;
+
+namespace k8s.Tests
+{
+    public class TokenFileAuthTests
+    {
+        [OperatingSystemDependentFact(Exclude = OperatingSystem.OSX | OperatingSystem.Windows)]
```

Comment:
Why is this excluded on OSX or Windows? This test should definitely work on all platforms.

Reply:
removed. i only had linux at hand to verify the test.